fairlearn/reductions/_moments/conditional_selection_rate.py

```diff
@@ -115,6 +114,10 @@ def signed_weights(self, lambda_vec):
         signed_weights = self.tags.apply(
             lambda row: adjust[row[_EVENT], row[_GROUP_ID]], axis=1
         )
+        # this flag is set if a `filter_value` was sent to :meth:`load_data()`
+        signed_weights = signed_weights.reindex(
+            list(range(self.dropped_X.shape[0] + self.X.shape[0])), fill_value=0
+        )
```

Comment:
Let's work with `tags` that don't have anything dropped. In that case you can maybe just do
```python
apply(lambda row: 0 if np.isnan(row[_EVENT]) else adjust[row[_EVENT], row[_GROUP_ID]])
```
and no need to reindex anything.

Reply:
Done.